lib/traject/extractors/crez_marc_kafka_extractor.rb

```diff
@@ -10,12 +10,20 @@ def initialize(reserves_file:, kafka:, topic:)
     @topic = topic
   end
 
+  # Scan through the Kafka topic and re-send any messages for records with CREZ data
   def process!
     t0 = Time.now
+    h = {}
```

Comment:
There may be some very clever way to read the topic backwards so we don't need to store a bunch of records all the time, but hopefully this whole file is very temporary and will go away within the month..

Reply:
🤞